docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+version: '3.4'
+
+services:
+    app:
+        build:
+            context: .
+            target: prod
```

Comment:
building 4 images with a Dockerfile was unexpected, but it works equivalently to having separate Dockerfiles

Reply:
Think it's a little easier too, don't have to run a Composer container every time FPM is started.

Could do with some separation with comments though.